pkgs/top-level/all-packages.nix

```
@@ -24574,20 +24574,20 @@ with pkgs;
 
   prospector = callPackage ../development/tools/prospector { };
 
-  protobuf = protobuf_24;
-
-  protobuf_25 = callPackage ../development/libraries/protobuf/25.nix {
-    abseil-cpp = abseil-cpp_202308;
-  };
-  protobuf_24 = callPackage ../development/libraries/protobuf/24.nix { };
-  protobuf_23 = callPackage ../development/libraries/protobuf/23.nix { };
-  protobuf_21 = callPackage ../development/libraries/protobuf/21.nix {
-    abseil-cpp = abseil-cpp_202103;
-  };
+  protobufVersions = lib.recurseIntoAttrs (import ../development/libraries/protobuf {
```

```suggestion
  protobufVersions = import ../development/libraries/protobuf {
```
Since we re-expose these at the top-level, we should probably avoid exposing them to hydra twice.